commands/__init__.py

```diff
@@ -0,0 +1,43 @@
+from .auto_indent_on_enter_between_curly_braces import TypescriptAutoIndentOnEnterBetweenCurlyBrackets
+from .error_info import TypescriptErrorInfo
+from .format import TypescriptFormatBrackets, TypescriptFormatDocument, TypescriptFormatLine, TypescriptFormatOnKey, \
```

Comment:
Several of these lines go way beyond the 79 char width recommended in PEP8.  You can use parens to group imports and not need the line continuation character.  For example, the above might look better as:

```python
from .format import (
    TypescriptFormatBrackets, 
    TypescriptFormatDocument, 
    TypescriptFormatLine, 
    TypescriptFormatOnKey,
    TypescriptFormatSelection, 
    TypescriptPasteAndFormat)
```